website/views.py

```diff
@@ -146,3 +146,17 @@ def search_result_by_booking_code(request):
         "website/search_result_by_booking_code.html",
         {"title": "Search Result By Booking Code", "reservations": reservation_detail},
     )
+
+
+def cancel_reservation(request, guest_id):
```

Comment:
Can we change this to just update_reservation so that updating and cancelling are handled using the same form? For both staff and guests.